sources/engine/Stride.Native/NativeInvoke.cs

```diff
@@ -43,7 +43,7 @@ internal class Module
         [ModuleInitializer]
         public static void Initialize()
         {
-            Core.Native.NativeInvoke.Setup();
+            
         }
```

Comment:
Probably fine to remove it entirely, no ?

Reply:
Good idea. I removed it and it seems to work anyway